kodilla-intro/src/main/java/Book.java

```diff
@@ -0,0 +1,18 @@
+public class Book {
+    private String author;
+    private String title;
+
+    public Book (String author, String title){
+        this.author = author;
+        this.title = title;
+    }
+
+    public static String of(String author, String title) {
+        Book book = new Book("Isaac Asimov", "The Galaxy");
```

Comment:
a jak bede chcial stworzyc ksiazke "Henryk Sienkiewicz", "W pustyni i w puszczy"?

Reply:
nie rozumiem pytania?? Wpiszesz zamiast "Isaac Asimow"==> "Henryk Sienkiewicz" a zamiast "The Galaxy" ==> "W pustyni i w puszczy" 
Zrobiłem na podstawie przykładu wskazanego w module.